src/ResourceManager/Network/Commands.Network/packages.config

```diff
@@ -19,8 +19,8 @@
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.4" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.4" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.5" targetFramework="net45" />
```

Comment:
Unfortunately, this change needs to be made across the board, for all modules, changing in one module will just cause conflicts.  WIlll discuss how to deal with this change in the mornign before cutting the new branch